app/code/community/Businessfactory/Roihuntereasy/Model/Cron.php

```diff
@@ -167,6 +126,157 @@ private function generateAndSaveFeed()
         return true;
     }
 
+    /**
+     * Feed generation function
+     */
+    private function generateAndSaveFeedCSV($products, $io) {
+        // CSV headers
+        $csvHeader = array(
+            'ID',
+            "Item title",
+            "Final URL",
+            "Image URL",
+            "Item description",
+            "Price",
+            "Sale price"
+        );
+
+        // write headers to CSV file
+        $io->streamWriteCsv($csvHeader);
+
+        $this->count = 0;
+        foreach ($products as $_product) {
+            switch ($_product->getTypeId()) {
+                case 'downloadable':
+                    if ($_product->getPrice() <= 0) {
+                        break;
+                    }
+//              Else same processing as simple product
+                case 'simple':
+                    $productDict = $this->get_simple_product_dict($_product);
+                    $io->streamWriteCsv($productDict);
+                    break;
+                case 'configurable':
+                    $productDictArray = $this->get_configurable_product_dict($_product);
+                    foreach ($productDictArray as $productDict){
+                        $io->streamWriteCsv($productDict);
+                    }
+                    break;
+            }
+        }
+
+    }
+
+    /**
+     * @param Mixed $_product
+     * @param XMLWriter $xmlWriter
+     */
+    private function get_configurable_product_dict($_product)
+    {
+        $conf = Mage::getModel('catalog/product_type_configurable')->setProduct($_product);
+        $simple_collection = $conf->getUsedProductCollection()->addAttributeToSelect('*')->addFilterByRequiredOptions();
+
+        $productArray = array();
+
+        foreach ($simple_collection as $_childproduct) {
+            $productDict = array(
+                "ID" => $this->get_id($_product, $_childproduct),
+                "Item title" => $this->get_title($_product),
+                "Final URL" => $this->get_product_url($_product),
+                "Image URL" => $this->get_image_url($_childproduct),
+                "Item description" => $this->get_description($_product),
+                "Price" => $this->get_price($_childproduct, true),
+                "Sale price" => $this->get_sale_price($_childproduct, true),
+            );
+            array_push($productArray, $productDict);
+        }
+        return $productArray;
+    }
+
+    /**
+     * @param $_product
+     * @param XMLWriter $xmlWriter
+     */
+    private function get_simple_product_dict($_product)
+    {
+        $productDict = array(
+            "ID" => $this->get_id($_product, null),
+            "Item title" => $this->get_title($_product),
+            "Final URL" => $this->get_product_url($_product),
+            "Image URL" => $this->get_image_url($_product),
+            "Item description" => $this->get_description($_product),
+            "Price" => $this->get_price($_product, true),
```

Comment:
Is this price formatted according to Google csv specification? You have to use ""."" as the decimal mark independent of shop local settings. 
From Google specification: 
> Number followed by the currency code, ISO 4217 standard. Use ""."" as the decimal mark.